docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -140,12 +153,19 @@ done
 
 *You should be finished!*
 
-* Check to make sure the cluster can see the minion (on fed-master).
+* Check to make sure the cluster can see the node on fed-master.
 
 ```
-kubectl get minions
+kubectl get nodes
 NAME                LABELS              STATUS
-fed-minion          <none>              Ready
+fed-node          name=fed-node-label     Ready
+```
+* Deletion of nodes.
+
+To delete _fed-node_ from your kubernetes cluster, run following on fed-master:
+
+```
+$ kubectl delete -f node.json
```

Comment:
I think it's ok to talk about delete, but you might want to call out explicitly that doing so means the next steps wouldn't work...